skyportal/tests/frontend/test_filters.py

```diff
@@ -0,0 +1,49 @@
+import uuid
+import pytest
+from baselayer.app.env import load_env
+
+
+_, cfg = load_env()
+
+
+@pytest.mark.flaky(reruns=2)
+def test_add_filter(driver, super_admin_user, user, public_group, public_stream):
+    driver.get(f'/become_user/{super_admin_user.id}')
+    driver.get('/groups')
+    driver.wait_for_xpath('//h6[text()="All Groups"]')
+    el = driver.wait_for_xpath(f'//a[contains(.,"{public_group.name}")]')
+    driver.execute_script("arguments[0].click();", el)
+    # add stream
+    driver.wait_for_xpath(f'//button[contains(.,"Add stream")]').click()
+    driver.wait_for_xpath('//input[@name="stream_id"]/..', timeout=10).click()
+    driver.wait_for_xpath(f'//li[contains(.,"{public_stream.id}")]', timeout=10)
+    stream = driver.switch_to.active_element
+    stream.click()
+    add_stream = driver.wait_for_xpath_to_be_clickable(f'//button[@type="submit"]')
+    driver.execute_script("arguments[0].click();", add_stream)
+
+    # add filter
+    filter_name = str(uuid.uuid4())
+    driver.wait_for_xpath_to_be_clickable(
+        f'//button[contains(.,"Add filter")]', timeout=10
+    )
+    flt = driver.switch_to.active_element
+    flt.click()
+    driver.wait_for_xpath(f'//button[contains(.,"Add filter")]').click()
```

Comment:
Consider using the driver.click_xpath() functions from baselayer where possible in this test instead? We had issues with MUI buttons before right? Wouldn't hurt I think.

Reply:
Yeah, seems like the test failed on Travis so this is probably a good idea.

Reply:
Switched to using `driver.click_xpath()` everywhere where it didn't cause a failure locally, will see what the CI has to say...

Reply:
@dannygoldstein discovered a nice trick using `.pause(2)` to allow for MUI transitions to finish, but I didn't get a chance to put it in before this got merged 😂 will do in a next pr ;-)